libs/deps/merch-card.js

```diff
@@ -906,6 +906,11 @@ merch-card[variant="mini-compare-chart"] .footer-row-cell-description a {
         font-size: var(--consonant-merch-card-body-xs-font-size);
     }
 
+    html[lang="he"] merch-card[variant="mini-compare-chart"] [is="inline-price"] .price-recurrence::before {
```

Comment:
Is this actually getting applied? When inspecting, no `::before` pseudo-element is rendered by the browser. Manually adding this rule makes it visible.

| Current result | Expected |
| ------------- | ------------- |
| <img width="373" alt="Screenshot 2024-06-25 at 12 21 34" src="https://github.com/adobecom/milo/assets/11267498/89ccbc42-da81-4292-aa4b-d7e0b3765270"> | <img width="270" alt="Screenshot 2024-06-25 at 12 21 46" src="https://github.com/adobecom/milo/assets/11267498/e398f531-79a3-44fa-9d67-5d773090727f"> |

Reply:
did you inspect it on a page with il_he locale? (e.g: https://main--cc--adobecom.hlx.live/il_he/creativecloud?milolibs=MWPW-151429--milo--adobecom)

![image](https://github.com/adobecom/milo/assets/330057/50e53ddc-0a43-4e5a-9543-829c3b0f1a8a)

Reply:
Yep, this is tested in incognito as well
<img width="672" alt="Screenshot 2024-06-25 at 15 12 53" src="https://github.com/adobecom/milo/assets/11267498/2024cee7-cfd8-48a4-a8a5-05f80b5d5cbf">

Reply:
on mobile?

Reply:
@overmyheadandbody you're screenshot appears to be from desktop viewport. Try inspecting in mobile, the change should appear:
![Screenshot 2024-06-25 at 6 51 30 AM](https://github.com/adobecom/milo/assets/1179549/3d5ed688-e2af-445d-9083-6cd813365f3b)

Reply:
Oh, it's just for mobile, right? With styles defined in JS it's difficult to grasp the media queries they relate to. Also, the PR title and description don't make any reference to this being a mobile-specific behavior. Again, it's difficult for non-merch folks to review merch PRs without enough context 🙂 Yes, it's there on mobile.